client/components/Editor/schemas/base.ts

```diff
@@ -7,34 +7,53 @@ export const baseNodes = {
 	},
 	paragraph: {
 		selectable: false,
+		// reactive: true,
 		content: 'inline*',
 		group: 'block',
 		attrs: {
+			id: { default: null },
```

Comment:
I should have thought of this earlier, but we should do some research into how adding IDs to paragraphs and other common block text nodes will affect the collaborative editing experience — for instance whether two clients attempting to open a paragraph in the same place will now open two paragraphs rather than having their changes merged into one. That could actually be a good thing! But I want to see what's up, so this is a note to myself to play around with it a little bit in a REPL.